CI_JobHistory.py

```diff
@@ -189,9 +195,13 @@ def display_ci_links(config_data):
         for ci_name in config_data.keys():
             j=j+1
             ci_name_list.append(ci_name)
+   
 
         selected_ci = config_vars.get('Settings', 'selected_ci')
         options_int_list = [int(value) for value in selected_ci.split(',')]
+        if len(ci_name_list)+1 in options_int_list:
```

Comment:
If we select All option, i guess we can just copy the entire config dict to `selected_config_data`, instead of iterating and creating a new dict and copying it over again.
In the beginning itself, check if all is selected, copy the dict and return it there itself, no need of any other checks i believe